client/src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import LumenLogo from '../../test.png';
+import classes from './Logo.module.css';
+
+const logo = () => (
+  <div className={classes.Logo}>
+    <img src={LumenLogo} alt="LumenLogo" />
```

Comment:
I would suggest wrapping up img tag as an independent component. I am sure that will help us in future development a lot

Reply:
Sure, actually img is wrapped in a Logo component but that is specific to our logo.